BioArchLinux/vsearch/lilac.yaml

```diff
@@ -0,0 +1,13 @@
+build_prefix: extra-x86_64
+maintainers:
+  - github: althonos
+    email: martin.larralde@embl.de
+pre_build_script: |
+  update_pkgver_and_pkgrel(_G.newver.lstrip('v'))
+post_build_script: |
+  git_pkgbuild_commit()
```

Comment:
Would you also consider update its AUR repo if you want to create one?

Reply:
Yes, I'll create an AUR repo as well.

Reply:
If you give a aur repo, add `BioArchLinuxBot` as co-maintainer in AUR, add `update_aur_repo()` to `post_build_script:`

Reply:
Done.